src/QsCompiler/Tests.Compiler/TestCases/LocalVerification.qs

```diff
@@ -1165,6 +1174,15 @@ namespace Microsoft.Quantum.Testing.LocalVerification {
     @ DeprecatedAttribute()
     newtype UsingDeprecatedAttribute3 = Unit;
 
+    @ Deprecated("")
+    function DeprecatedAttributeInDeprecatedCallable() : Unit {
+        UsingDeprecatedAttribute1()
+    }
+
+    function UsingDepAttrInDepCall() : Unit {
+        DeprecatedAttributeInDeprecatedCallable()
```

Comment:
```suggestion
        DeprecatedAttributeInDeprecatedCallable();
```

I believe this is the cause of the test failure. :) I figured this out by noticing that the test output says there was an unexpected error code 2003. The error codes are defined in DataStructures/Diagnostics.fs. 2003 is ExpectingSemicolon. https://github.com/microsoft/qsharp-compiler/blob/90fcb6298f6d00f632129699b70cc2de19cb72e2/src/QsCompiler/DataStructures/Diagnostics.fs#L16